apps/api-extractor/package.json

```diff
@@ -48,7 +48,7 @@
     "resolve": "~1.17.0",
     "semver": "~7.3.0",
     "source-map": "~0.6.1",
-    "typescript": "~4.7.4"
+    "typescript": "~4.8.4"
```

Comment:
I checked the TS4.8 release notes and didn't find anything that looked like it could be a problem for API Extractor 👍